### ISSUE TITLE
Comet Cache Pro generating WSOD for logged-out users

### ISSUE THREAD

**Issue by KTS915** [wpsharks/comet-cache#802]
This isn't happening on every site, but I have two sites where this has happened. It is not affecting logged-in users, but that might be because I have it set not to cache for logged-in users on these sites.

Yesterday, and running version 160706,  debug reported a problem with `advanced-cache.php` and I got everything up and running again by deactivating CC and then deleting that file.

But today, on another site, and running version 160709, that isn't working and I'm not getting any helpful debug messages other than a 500 error. But, as soon as I deactivate CC, the site is back up.

**Comment by KTS915** [comment]
OK, now it's happened to all the sites using CC Pro. Just a WSOD when logged out.

**Comment by KTS915** [comment]
I've been doing some further investigation. The culprit seems to be within the HTML compressor, specifically combining CSS from head and body into fewer files. Unless something has changed in the plugin, that's surprising, because I've been using that option for a long time. But I'll keep digging and see what else I can discover.

**Comment by KTS915** [comment]
Yes, turning off the setting to combine CSS from head and body into fewer files has solved the problem.

But there was one oddity. One site was reporting in View Page Source that pages were being cached even though Comet Cache Pro was actually turned off. I don't know if that was just a one-off anomaly, or whether it might be related.

**Comment by raamdev** [comment]
@KTS915 It sounds like you might have something in the `.htaccess` file that is rewriting the requests and serving cache files, which is odd because Comet Cache does not have such a feature.

Nothing has changed in the HTML Compressor in the past several releases, so an issue there does indeed seem very odd.

The newest Comet Cache feature is Apache Optimizations, which has options to tweak the `.htaccess` file, but Comet Cache will not touch the `.htaccess unless you specifically enable some of those Apache Optimizations.

**Comment by KTS915** [comment]
Hmm, thanks, @raamdev ! 

I have now followed up your suggestion and found nothing in my .htaccess files other than the regular WP stuff and things put there by CC. But I think you must be on the right track just the same.

As I was investigating, I also tried re-enabling the combining feature in the HTML compressor. This time I found that the initial 500 error first appeared, but was then quickly replaced by the proper appearance of each site. So my guess now is that my host has been doing some maintenance. But there was a new oddity: the page notes indicated that caching was heppening, while HTML compression was not, even though that was turned on. But disabling the combining feature in the HTML compressor has meant that HTML compression is happening again!

So I think it's an issue with my host, and I'm going to close this. Thanks!
